docs/source/guide/install_enterprise_k8s.md

```diff
@@ -0,0 +1,418 @@
+---
+title: Deploy Label Studio Enterprise on Kubernetes
+badge: <i class='ent'></i>
+type: guide
+order: 203
+meta_title: Deploy Label Studio Enterprise on Kubernetes
+meta_description: Deploy Label Studio Enterprise on Kubernetes, such as on Amazon Elastic Container Service for Kubernetes, to create machine learning and data science projects in a scalable containerized environment. 
+---
+
+Deploy Label Studio Enterprise on a Kubernetes Cluster using Helm 3. You can use this Helm chart to set up Label Studio Enterprise for deployment onto a Kubernetes cluster and install, upgrade, and manage the application. 
+
+> Beta documentation: Label Studio Enterprise v2.0.0 is currently in Beta. As a result, this documentation might not reflect the current functionality of the product.
+
+<div class="enterprise"><p>
+To install Label Studio Community Edition, see <a href="install.html">Install and Upgrade Label Studio</a>. This page is specific to the Enterprise version of Label Studio.
+</p></div>
+
+## Required software prerequisites
+
+- Kubernetes and kubectl version 1.17 or higher
+- Helm version 3.6.3 or higher
+- Redis version 6.0.5 or higher
+- PostgreSQL version 11.9 or higher
+
+This chart has been tested and confirmed to work with the [NGINX Ingress Controller](https://www.nginx.com/products/nginx-ingress-controller/) and [cert-manager](https://cert-manager.io/docs/).
```

Comment:
That the link to a paid version of nginx-ingress, pls replace it with OSS version: https://kubernetes.github.io/ingress-nginx/

Reply:
oops, thanks fixing!